renku/core/management/repository.py

```diff
@@ -417,3 +419,42 @@ def init_repository(self, name=None, force=False):
             metadata.updated = datetime.now(timezone.utc)
 
         return str(path)
+
+    def init_empty_repository(self, force=False):
+        """Initialize an empty Renku repository."""
+        from git import Repo
+
+        if self.repo is not None and not force:
+            raise FileExistsError(self.repo.git_dir)
+
+        path = self.path.absolute()
+        self.repo = Repo.init(str(path))
+
+        return str(path)
+
+    def import_from_template(self, template_path, metadata, force=False):
+        """Render template files from a template directory."""
+        # adding author info? Creator.from_git(self.repo)
```

Comment:
That line in `init_repository` is just a hacky way to force a check on the git configuration. The "creator" is just the person making the initial commit so it's not necessary to add it anywhere manually.